packages/core/src/charts/combo.ts

```diff
@@ -0,0 +1,123 @@
+// Internal Imports
+import { AxisChart } from "../axis-chart";
+import * as Configuration from "../configuration";
+import {
+	ChartConfig,
+	ComboChartOptions,
+	ChartTypes,
+	Skeletons
+} from "../interfaces/index";
+import { Tools } from "../tools";
+
+// Components
+import {
+	Grid,
+	GroupedBar,
+	SimpleBar,
+	StackedBar,
+	Line,
+	TwoDimensionalAxes,
+	ZeroLine,
+	Scatter,
+	StackedScatter,
+	Area,
+	StackedArea,
+	Ruler,
+	StackedRuler,
+	// the imports below are needed because of typescript bug (error TS4029)
+	Legend,
+	LayoutComponent,
+	Component,
+	Skeleton
+} from "../components/index";
+
+const graphComponentsMap = {
+	[ChartTypes.LINE]: [Line, Scatter],
+	[ChartTypes.SCATTER]: [Scatter],
+	[ChartTypes.AREA]: [Area, Line, Scatter],
+	[ChartTypes.STACKED_AREA]: [StackedArea, Line, StackedScatter],
+	[ChartTypes.SIMPLE_BAR]: [SimpleBar],
+	[ChartTypes.GROUPED_BAR]: [GroupedBar, ZeroLine],
+	[ChartTypes.STACKED_BAR]: [StackedBar, StackedRuler]
+};
+
+export class ComboChart extends AxisChart {
+
+	constructor(holder: Element, chartConfigs: ChartConfig<ComboChartOptions>) {
+		super(holder, chartConfigs);
+
+		// Merge the default options for this chart
+		// With the user provided options
+		const chartOptions = Tools.mergeDefaultChartOptions(
+			Configuration.options.comboChart,
+			chartConfigs.options
+		);
+
+		// Warn user if no chartTypes defined
+		// Use skeleton chart instead
+		if (!chartConfigs.options.chartTypes) {
+			console.warn("No chartTypes defined for the Combo Chart!");
```

Comment:
if we consider this a potentially broken chart, then a console.error, if not, console.warn is fine

Reply:
updated 👍